docs/projects/rot-13/index.md

```diff
@@ -0,0 +1,64 @@
+---
+title: ROT-13 in Every Language
+layout: default
+date: 2018-11-20
+last-modified: 2018-11-20
+featured-image:
+tags: [rot-13]
+authors:
```

Comment:
Thanks for taking care of this! Would you mind adding yourself to the [`authors.yml`](https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml) file? That way this has a name to publish under.

Reply:
Happy to help! I have added myself.